src/proteus/interior/wrapper.py

```diff
@@ -45,21 +43,23 @@ def determine_interior_radius(dirs:dict, config:Config, hf_all:pd.DataFrame, hf_
     hf_row["gravity"] = 9.81
 
     # Target mass
-    M_target = config.struct.mass * M_earth
+    M_target = config.struct.mass_tot * M_earth
 
-    # We need to solve for the state M_int = config.struct.mass
+    # We need to solve for the state hf_row[M_tot] = config.struct.mass_tot
     # This function takes R_int as the input value, and returns the mass residual
     def _resid(x):
         hf_row["R_int"] = x
 
         log.debug("Try R = %.2e m = %.3f R_earth"%(x,x/R_earth))
 
+        # Use interior model to get dry mass from radius
         run_interior(dirs, config, IC_INTERIOR, hf_all, hf_row, verbose=False)
         if solve_g:
```

Comment:
Is there a case `solve_g` would be meaningfully turned off?

Reply:
No, I don't think so. Leaving it set to true works entirely fine for both Dummy and SPIDER.

The reason this flag is here is because Aragog will crash when it is turned on - I think related to this issue with density that @planetmariana is working on. We should try to remove this as soon as possible.